examples/mean-reversion.go

```diff
@@ -0,0 +1,241 @@
+package main
+
+import (
+	"fmt"
+	"math"
+	"os"
+	"time"
+
+	"github.com/alpacahq/alpaca-trade-api-go/alpaca"
+	"github.com/alpacahq/alpaca-trade-api-go/common"
+	"github.com/shopspring/decimal"
+)
+
+type alpacaClientContainer struct {
+	client         *alpaca.Client
+	runningAverage float64
+	lastOrder      string
+	timeToClose    float64
+	stock          string
+}
+
+var alpacaClient alpacaClientContainer
+
+func init() {
+	API_KEY := "YOUR_API_KEY_HERE"
+	API_SECRET := "YOUR_API_SECRET_HERE"
+	BASE_URL := "https://paper-api.alpaca.markets"
+
+	os.Setenv(common.EnvApiKeyID, API_KEY)
+	os.Setenv(common.EnvApiSecretKey, API_SECRET)
+	alpaca.SetBaseUrl(BASE_URL)
+
+	alpacaClient = alpacaClientContainer{
+		alpaca.NewClient(common.Credentials()),
+		0.0,
+		"",
+		0.0,
+		"AAPL",
+	}
+}
+
+func main() {
+	// First, cancel any existing orders so they don't impact our buying power.
+	status, until, limit := "open", time.Now(), 100
+	orders, _ := alpacaClient.client.ListOrders(&status, &until, &limit)
+	for _, order := range orders {
+		_ = alpacaClient.client.CancelOrder(order.ID)
+	}
+
+	// Wait for market to open
+	fmt.Println("Waiting for market to open...")
+	for {
+		isOpen := alpacaClient.awaitMarketOpen()
+		if isOpen {
+			break
+		}
+		time.Sleep(60000 * time.Millisecond)
+	}
+	fmt.Println("Market Opened.")
+
+	// Wait until 20 bars of data since market open have been collected.
+	fmt.Println("Waiting for 20 bars...")
```

Comment:
Would be nice to use variable for 20 and replace all use of constant for this

Reply:
Good idea.  Implemented